### PR TITLE
DotNet support for base64 operations in ImageTools

### PR DESCRIPTION
**Pull request by eeue56**
Now this will work on DotNet platforms: 

```
<App>
	<JavaScript>
		var ImageTools = require("FuseJS/ImageTools");
		var Observable = require("FuseJS/Observable");

		var imagePath = Observable();
		var base64Image =	"iVBORw0KGgoAAAANSUhEUgAAAA8AAAAPAQMAAAABGAcJAAAABlBMVEX9//wAAQATpOzaAAAAH0l" +
							"EQVQI12MAAoMHIFLAAYSEwIiJgYGZASrI38AAAwBamgM5VF7xgwAAAABJRU5ErkJggg==";
		ImageTools.getImageFromBase64(base64Image)
		.then(function(image) {
			imagePath.value = image.path;
			ImageTools.getBase64FromImage(image).then(function(b64){
				console.log(b64);
			});
		});

		module.exports = { test: new Date().toString(), image: imagePath };
	</JavaScript>
	<Image File="{image}" />
</App>
```

Previously this was unsupported. This is a first PR to add full DotNet compatibility for image tools

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,6 +55,7 @@
 
 ## ImageTools
 - Changed the algorithm for creating new file names for temporary images. Previously this used a date format that caused problems when several images were created in sub-second intervals, breaking custom map marker icons, for instance.
+- Added supported for encoding/decoding images to/from base64 on DotNet platforms, including Windows and Mac OS X.
```
Comment (kusma) [inline CHANGELOG.md]: This is in the wrong section of the changelog by now

**Comment by kusma** [comment]
@eeue56: Any plans of getting those tests out of "wip"-status?

**Comment by eeue56** [comment]
Yep, it's next on my TODO list

**Comment by eeue56** [comment]
moved to https://github.com/fusetools/fuselibs-public/pull/30

**Pull request closed without merging**